src/preprocessing/passes/ackermann.cpp

```diff
@@ -35,8 +34,7 @@ namespace passes {
 
 /* -------------------------------------------------------------------------- */
 
-namespace
-{
+namespace {
```

Comment:
This change seems unrelated and does not conform to our style. Please revert.

Reply:
This is changed by clang-format, and I see that other files in cvc4 are using the style with connected {. Are you sure to revert this?

Reply:
Oh yes, you are right. It is actually matching our style (we have the `{` on separate line for other constructs in the language). However, we usually try to only format the changes and not the whole file (that way, we are slowly changing old files slowly to the new formatting style instead of having a huge commit changing the style in all our files). You can use `git clang-format <commit>` to format the diff between the current state and `<commit>`. For example, if your branch is up-to-date with `master`, you should be able to do `git clang-format master` and it should only format the changes that you made.

Reply:
OK, thanks for pointing out.